sirepo/package_data/template/activait/loadImages.py.jinja

```diff
@@ -56,35 +33,60 @@ class SirepoHDF5Sequence(keras.utils.Sequence):
         self.batch_size = batch_size
         self.shuffle = shuffle
         self.return_value = None
+        self.inScaler = self._scaler(in_scaler)
+        self.outScaler = self._scaler(out_scaler)
+        self._fit()
 
     def set_return_value(self, name):
         self.return_value = name
         return self
 
+    def _scaler(self, scaler):
+        if scaler is None:
+            return
+        if scaler == MinMaxScaler:
+            return scaler(feature_range=({{ feature_min }}, {{ feature_max }}))
+        return scaler()
+
+    def _fit(self):
+        if not self.inScaler and not self.outScaler:
+            return
+        with h5py.File(self.filename, "r", libver="latest", swmr=True) as f:
+            self.original_shape_x = f[self.x.key].shape
```

Comment:
We don't need the extra libver and swmr settings for both the h5py.File() calls - the file isn't being written while the readers are running.